packages/core-api/src/versions/2/blocks/methods.ts

```diff
@@ -50,7 +50,7 @@ const search = async request => {
 export function registerMethods(server) {
   server.method("v2.blocks.index", index, {
     cache: {
-      expiresIn: 8 * 1000,
+      expiresIn: 4 * 1000,
```

Comment:
I would set it to 6 seconds, that still gives 2 seconds room.

Reply:
should be enough, i agree.